### PR TITLE
fix: cssref -> CSSRef

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

File: files/en-us/web/css/alternative_style_sheets/index.html [hunk 1]
```diff
@@ -32,7 +32,7 @@ <h2 id="An_example_specifying_the_alternative_stylesheets">An example: specifyin
 
 <h3 id="Try_it_out">Try it out</h3>
 
-<p><a href="/samples/cssref/altstyles/index.html">Click here</a> for a working example you can try out.</p>
+<p><a href="/samples/CSSRef/altstyles/index.html">Click here</a> for a working example you can try out.</p>
```
Comment (rachelandrew) [inline files/en-us/web/css/alternative_style_sheets/index.html:35]: `<p><a href="/samples/cssref/altstyles/index.html">Click here</a> for a working example you can try out.</p>`

Comment (rachelandrew) [inline files/en-us/web/css/alternative_style_sheets/index.html:35]:
```suggestion
<p><a href="/samples/cssref/altstyles/index.html">Click here</a> for a working example you can try out.</p>
```

File: files/en-us/web/css/element()/index.html [hunk 2]
```diff
@@ -31,7 +31,7 @@ <h2 id="Syntax">Syntax</h2>
 
 <h2 id="Examples">Examples</h2>
 
-<p>These examples can be <a href="/samples/cssref/moz-element.html">viewed live</a> in builds of Firefox that support <code>-moz-element()</code>.</p>
+<p>These examples can be <a href="/samples/CSSRef/moz-element.html">viewed live</a> in builds of Firefox that support <code>-moz-element()</code>.</p>
```
Comment (rachelandrew) [inline files/en-us/web/css/element()/index.html:34]:
```suggestion
<p>These examples can be <a href="/samples/cssref/moz-element.html">viewed live</a> in builds of Firefox that support <code>-moz-element()</code>.</p>
```

File: files/en-us/web/css/-webkit-box-reflect/index.html [hunk 3]
```diff
@@ -66,7 +66,7 @@ <h2 id="Browser_compatibility">Browser compatibility</h2>
 <h2 id="See_also">See also</h2>
 
 <ul>
- <li>The Apple <a class="external" href="http://developer.apple.com/library/safari/documentation/appleapplications/reference/safaricssref/Articles/StandardCSSProperties.html#//apple_ref/doc/uid/TP30001266-SW16">documentation</a>.</li>
+ <li>The Apple <a class="external" href="http://developer.apple.com/library/safari/documentation/appleapplications/reference/safariCSSRef/Articles/StandardCSSProperties.html#//apple_ref/doc/uid/TP30001266-SW16">documentation</a>.</li>
```
Comment (rachelandrew) [inline files/en-us/web/css/-webkit-box-reflect/index.html:69]:
```suggestion
 <li>The Apple <a class="external" href="http://developer.apple.com/library/safari/documentation/appleapplications/reference/safaricssref/Articles/StandardCSSProperties.html#//apple_ref/doc/uid/TP30001266-SW16">documentation</a>.</li>
```

**Review by rachelandrew — CHANGES_REQUESTED** [review]
There were a few URLs caught up in this that we probably don't want to change.

**Comment by nschonni** [comment]
Sorry about that, I re-ran the find/replace with `{{CSSRef` so it won't catch the URL stuff

**Review by rachelandrew — APPROVED** [review]
thanks!

**Pull request merged**